Day-6-7/src/index.css

```diff
@@ -22,13 +22,12 @@ body {
   align-items: center;
   flex-wrap: wrap;
   gap: 15px;
-
   padding: 20px;
 }
 
 .card-tag {
   border: 1px solid #fff;
```

Comment:
Now your app is growing, can add styles to modular CSS.

Reply:
Yeah I will do it using tailwind now